### ISSUE TITLE
Add .env in .gitignore

### ISSUE THREAD

**Issue by roberto-butti** [roberto-butti/fit-geojson-vuejs#96]
Exclude .env file from git.
You need to add into .gitignore file the .env (we already ignore some .env.*)

**Comment by AzimPathan** [comment]
What exactly needs to be done?

**Comment by roberto-butti** [comment]
> What exactly needs to be done?

in .gitignore file there are these lines:

```
# local env files
.env.local
.env.*.local
```

we need to and also .env :

```
# local env files
.env
.env.local
.env.*.local
```

**Comment by roberto-butti** [comment]
fixed by PR #99
